test/functional/test_framework/wallet.py

```diff
@@ -142,20 +142,29 @@ def scan_tx(self, tx):
                 self._utxos.append(self._create_utxo(txid=tx["txid"], vout=out["n"], value=out["value"], height=0))
 
     def sign_tx(self, tx, fixed_length=True):
-        """Sign tx that has been created by MiniWallet in P2PK mode"""
-        assert_equal(self._mode, MiniWalletMode.RAW_P2PK)
-        (sighash, err) = LegacySignatureHash(CScript(self._scriptPubKey), tx, 0, SIGHASH_ALL)
-        assert err is None
-        # for exact fee calculation, create only signatures with fixed size by default (>49.89% probability):
-        # 65 bytes: high-R val (33 bytes) + low-S val (32 bytes)
-        # with the DER header/skeleton data of 6 bytes added, this leads to a target size of 71 bytes
-        der_sig = b''
-        while not len(der_sig) == 71:
-            der_sig = self._priv_key.sign_ecdsa(sighash)
-            if not fixed_length:
-                break
-        tx.vin[0].scriptSig = CScript([der_sig + bytes(bytearray([SIGHASH_ALL]))])
-        tx.rehash()
+        """Sign tx that has been created by MiniWallet in P2TR/RAWSCRIPT/P2PK mode"""
```

Comment:
in 0c20a83fa658fc660b138382d907615b9a60b845: Any reason to duplicate the list of modes, given that it just repeats all modes?

Reply:
Should I remove the comment or replace it with "Sign tx that has been created by MiniWallet"?